doc/build-instructions/Build_Instructions_V11.md

```diff
@@ -168,9 +168,8 @@ bash get_source.sh
 :penguin:
 When you have all the source files that you need, run the configure script, which detects how to build in the current build environment.
 ```
-bash configure --with-freemarker-jar=/<my_home_dir>/freemarker.jar
```

Comment:
The `<my_home_dir>` matches step 3 in `Setting up your build environment without Docker`, I don't think it should removed. We could break this up into two command lines 'With Docker' and `Without Docker`, or change it to `<my_home_dir or /root when using Docker>`.

Reply:
I that case, https://github.com/eclipse/openj9/blob/master/doc/build-instructions/Build_Instructions_V8.md is wrong, because this is where I got the /root/freemaker.jar path (which was perfect for me since I used the Docker option).

Reply:
Anyway @pshipton what you propose sounds good, but I should apply it to all build instruction documents, right?

Newbie question: should I cancel my pull request and re-create a branch re-roll a new PR ?

Reply:
Yes, we should change all versions to match. You don't need to cancel the pull request, you can force push a new change to your branch.

Reply:
Hi @pshipton, I was getting ready to fix build instructions as agreed above, but I think I found another glitch: shouldn't `/<my_home_dir>` actually be `<my_home_dir>`, everywhere it is used in Linux build instructions?

Indeed, let us suppose that `<my_home_dir>` is `/u/users/vermeera`, then `/<my_home_dir>` becomes `//u/users/vermeera`. While Linux is tolerant and considers `//` to be equivalent to `/`, this sounds like a typo. 

Would you agree that I should also fix this in the same PR ?

Reply:
Makes sense.

Reply:
Done, and this time I have successfully "squashed" the commits on this branch into one.
Should I redo PR request or is that automatic ?

Reply:
You don't need to redo anything related to your PR, just update your branch. I still see 2 commits, so they need to be squashed. Java 11 updates look good, but I only see a partial update for Java 13 and the leading `/` isn't removed. No update for Java 8.